openmrs/omod/src/main/java/org/openmrs/projectbuendia/webservices/rest/OrderResource.java

```diff
@@ -91,74 +87,57 @@
     supportedOpenmrsVersions = "1.10.*,1.11.*"
 )
 public class OrderResource implements Listable, Searchable, Retrievable, Creatable, Updatable {
-    static final User CREATOR = new User(1);  // fake value
-    static final RequestLogger logger = RequestLogger.LOGGER;
-    static Log log = LogFactory.getLog(OrderResource.class);
-    static final String FREE_TEXT_ORDER_UUID = "buendia-concept-free_text_order";
+    private static final User CREATOR = new User(1);  // fake value
+    private static final RequestLogger logger = RequestLogger.LOGGER;
+    private static Log log = LogFactory.getLog(OrderResource.class);
+    private static final String FREE_TEXT_ORDER_UUID = "buendia-concept-free_text_order";
 
-    final PatientService patientService;
-    final OrderService orderService;
-    final ProviderService providerService;
-    final ConceptService conceptService;
-    final EncounterService encounterService;
+    private final PatientService patientService;
+    private final OrderService orderService;
+    private final ProviderService providerService;
+    private final EncounterService encounterService;
+    private final ProjectBuendiaService buendiaService;
 
     public OrderResource() {
         patientService = Context.getPatientService();
         orderService = Context.getOrderService();
         providerService = Context.getProviderService();
-        conceptService = Context.getConceptService();
         encounterService = Context.getEncounterService();
+        buendiaService = Context.getService(ProjectBuendiaService.class);
     }
 
-    @Override public SimpleObject getAll(RequestContext context) throws ResponseException {
-        try {
-            logger.request(context, this, "getAll");
-            SimpleObject result = getAllInner();
-            logger.reply(context, this, "getAll", result);
-            return result;
-        } catch (Exception e) {
-            logger.error(context, this, "getAll", e);
-            throw e;
-        }
+    @Override
+    public SimpleObject getAll(RequestContext context) throws ResponseException {
+        return search(context);
     }
 
-    private SimpleObject getAllInner() throws ResponseException {
-        return getSimpleObjectWithResults(getAllOrders());
-    }
+    private SimpleObject handleSync(RequestContext context) throws ResponseException {
+        Date date = RequestUtil.mustParseSyncFromDate(context);
 
-    SimpleObject getSimpleObjectWithResults(Collection<Order> orders) {
-        List<SimpleObject> jsonResults = new ArrayList<>();
+        // Set the next sync from time to be one second in the past because there's issues
+        // due to OpenMRS rounding truncating insertion times. See
+        // https://github.com/projectbuendia/client/pull/90
+        Date nextSyncFrom = new Date(System.currentTimeMillis() - 1000);
```

Comment:
Yep! That's how I worked out the tests were failing in https://github.com/projectbuendia/client/pull/153 :)